semgrep/semgrep/cli.py

```diff
@@ -393,6 +446,7 @@ def __validate_lang(option: str, lang: Optional[str]) -> str:
     # help="WARNING: allow rules to run arbitrary code (pattern-where-python)",
 )
 def cli(
+    *,
```

Comment:
What does this do?

Reply:
https://www.python.org/dev/peps/pep-3102/